src/app/pages/send-crypto-asset/hooks/use-fees.ts

```diff
@@ -0,0 +1,49 @@
+import { useMemo } from 'react';
+
+import { useBitcoinFees } from '@app/query/bitcoin/fees/fee-estimates.hooks';
+import { useStacksFees } from '@app/query/stacks/fees/fees.hooks';
+import {
+  useFtTokenTransferUnsignedTx,
+  useStxTokenTransferUnsignedTxState,
+} from '@app/store/transactions/token-transfer.hooks';
+
+import {
+  getEstimatedUnsignedStacksTxByteLength,
+  getSerializedUnsignedStacksTxPayload,
+} from '../form-utils';
+import { useStacksFtParams } from '../forms/stx-sip10/use-stacks-ft-params';
+
+export function useBtcFees() {
+  // TODO: Replace hardcoded number here (200) with the tx byte length
+  const { data: btcFees } = useBitcoinFees(200);
+  return btcFees;
+}
```

Comment:
This perpetuates the pattern of wrapping react query so that the inner properties are inaccessible. What if we want to hook into the loading state later?

Suggest changing this so we use `useBitcoinFees` directly

Reply:
Also interested how bitcoin fees work. Are they also calculated from tx byte size? If so, how does this vary transaction to transaction if we only support single sig, single recipient transfers. If this is fixed, perhaps we can hard code rather than calculate tx size, for now

Reply:
Yeah, true, was just trying to pull stuff out of component to make it less crowded. I'll rethink it ...prob not nec.

Reply:
> Also interested how bitcoin fees work. Are they also calculated from tx byte size?

Yes, that is why it is hardcoded here as `200` until we implement.